src/getRevision.sh

```diff
@@ -1,5 +1,13 @@
 SCRIPT_DIRECTORY=`dirname $0`
 rev=\"`git rev-parse --short HEAD`\"
+lastrev=$(cat $SCRIPT_DIRECTORY/revision.txt)
```

Comment:
It is a clumsy solution.
We already have current revision in Revision.h. It can be parsed as
sed -n '1{p;q}' $SCRIPT_DIRECTORY/Revision.h | awk -F'"' {'print $2'}

Reply:
or 
head -n 1 $SCRIPT_DIRECTORY/Revision.h | awk -F'"' {'print $2'}

Reply:
Yes, you are right, scripting is not my forte all I was just going with something that worked for me. I can update it to the second option.

Reply:
This command produces the revision with double quotes:
```git rev-parse --short HEAD```
so I had to change your command to this and then it works:
```head -n 1 $SCRIPT_DIRECTORY/Revision.h | awk -F'PLUGIN_REVISION ' {'print $2'}```